static-site/components/list-resources/index.tsx

```diff
@@ -315,7 +315,7 @@ function Filter({
   return (
     <div className="filter">
       <Select
-        placeholder={`Filter by keywords in ${resourceType==='dataset'?'dataset':'file'} names`}
+        placeholder={`Filter by keywords in ${displayResourceType(resourceType)} names`}
```

Comment:
Should make a similar change in a couple places within resource-groups.tsx, e.g.

https://github.com/nextstrain/nextstrain.org/blob/58a37da8401831da63474841d927167e68bc9545/static-site/components/list-resources/resource-group.tsx#L186

Also the resource-groups.tsx just has a couple places that currently hardcodes the word "datasets", e.g. 

https://github.com/nextstrain/nextstrain.org/blob/58a37da8401831da63474841d927167e68bc9545/static-site/components/list-resources/resource-group.tsx#L246

Reply:
Done as separate commits: eccae57b5cef063feb80188bc20712db08122afa...44f3a9dc0bf568f313f334ae29eb30c8c758e1a0

There's more places where "dataset" and "pathogen" are hardcoded, but they aren't shown for unversioned resource listings. I've left those as-is – it should be obvious what needs fixing if/when version info is added to more pages. I don't think I've missed anything that could appear with this PR's changes, but let me know if you find anything!